packages/core/core/src/requests/ConfigRequest.js

```diff
@@ -100,12 +106,48 @@ export async function loadPluginConfig<T: PluginWithLoadConfig>(
   }
 }
 
+const packageKeyCache = createBuildCache();
+
+export async function getPackageKeyContentHash(
+  filePath: ProjectPath,
+  packageKey: string,
+  options: ParcelOptions,
+): Async<string> {
+  let cacheKey = `${fromProjectPathRelative(filePath)}:${packageKey}`;
+  let cachedValue = packageKeyCache.get(cacheKey);
+
+  if (cachedValue) {
+    return cachedValue;
+  }
+
+  let conf = await readConfig(
+    options.inputFS,
+    fromProjectPath(options.projectRoot, filePath),
+  );
+
+  if (conf == null || conf.config[packageKey] == null) {
```

Comment:
The `packageKey` being removed could be a valid scenario couldn't it? Like if I remove custom config for a transformer that I no longer need because the defaults are file.

Reply:
Ahh yeah you're right, good catch.